mmedit/apis/video_interpolation_inference.py

```diff
@@ -192,7 +192,6 @@ def video_interpolation_inference(model,
         if start_index + lenth_per_step >= end_idx:
             break
 
-    print()
```

Comment:
I think it is used for creating a new line, instead of temporal debug. We may need it here.

Reply:
done